docker-compose.besu.yml

```diff
@@ -0,0 +1,33 @@
+version: "3.8"
+
+services:
+  besu:
+    image: hyperledger/besu:$BESU_VERSION
```

Comment:
i think this variable is missing from the .env.example
would be nice to add a reference to it here